subworkflows/local/amp.nf

```diff
@@ -33,24 +33,28 @@ workflow AMP {
     // AMPLIFY
     if ( !params.amp_skip_amplify ) {
         AMPLIFY_PREDICT ( ch_faa_for_amplify, [] )
-        ch_versions = ch_versions.mix(AMPLIFY_PREDICT.out.versions)
-        ch_ampresults_for_ampcombi = ch_ampresults_for_ampcombi.mix(AMPLIFY_PREDICT.out.tsv)
+        ch_versions                 = ch_versions.mix(AMPLIFY_PREDICT.out.versions)
+        ch_ampresults_for_ampcombi  = ch_ampresults_for_ampcombi.mix(AMPLIFY_PREDICT.out.tsv)
     }
 
     // MACREL
     if ( !params.amp_skip_macrel ) {
         MACREL_CONTIGS ( contigs )
-        ch_versions = ch_versions.mix(MACREL_CONTIGS.out.versions)
-        GUNZIP_MACREL ( MACREL_CONTIGS.out.amp_prediction )
-        ch_versions = ch_versions.mix(GUNZIP_MACREL.out.versions)
-        ch_ampresults_for_ampcombi = ch_ampresults_for_ampcombi.mix(GUNZIP_MACREL.out.gunzip)
+        ch_versions                 = ch_versions.mix(MACREL_CONTIGS.out.versions)
+        GUNZIP_MACREL_PRED ( MACREL_CONTIGS.out.amp_prediction )
+        GUNZIP_MACREL_ORFS ( MACREL_CONTIGS.out.all_orfs )
+        ch_versions                 = ch_versions.mix(GUNZIP_MACREL_PRED.out.versions)
+        ch_versions                 = ch_versions.mix(GUNZIP_MACREL_ORFS.out.versions)
+        ch_ampresults_for_ampcombi  = ch_ampresults_for_ampcombi.mix(GUNZIP_MACREL_PRED.out.gunzip)
+        ch_macrel_faa               = ch_macrel_faa.mix(GUNZIP_MACREL_ORFS.out.gunzip)
+        ch_faa_for_ampcombi         = ch_faa_for_ampcombi.mix(ch_macrel_faa)
     }
 
```

Comment:
Why did you choose to first create the `ch_macrel_faa`? For readability?
Wouldn't this have had the same effect (two lines less code): 
`ch_faa_for_ampcombi = ch_faa_for_ampcombi.mix(GUNZIP_MACREL_ORFS.out.gunzip)` ?